services/templeton/src/logcabin/logcabin.go

```diff
@@ -0,0 +1,76 @@
+package logcabin
```

Comment:
This package has not been updated for several iterations of changes by @johnworth and @strootman. Since it works, for now I haven't updated it, but it's on my roadmap a bit after the elasticsearch index wrangling process.